File: Client/HextechFriendsClient/Protocol/Client/RequestJoinTS.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HextechFriendsClient.Protocol.Client
{
    class RequestJoinTS
    {
        public string opCode = "requestJoinTS";
        public int level { get; set; }
        public int iconId { get; set; }
        public string lobbyId { get; set; }
        public string summonerName { get; set; }
        public string platform { get; set; }
        public string summonerId { get; set; }
    }
}

File: Server/src/main/java/xyz/sleepylux/hextechfriends/models/DTO/client/RequestJoinTS.java
package xyz.sleepylux.hextechfriends.models.DTO.client;

public class RequestJoinTS {
    private String opCode;
    private int level;
    private int iconId;
    private String lobbyId;
    private String summonerName;
    private String platform;
    private String summonerId;

    public int getIconId() { return iconId; }

    public int getLevel() { return level; }

    public String getLobbyId() { return lobbyId; }

    public String getSummonerName() { return summonerName; }

    public String getPlatform() { return platform; }

    public String getSummonerId() { return summonerId; }
}
